### DISCUSSION TITLE
New domain name review issue

### DISCUSSION THREAD

**Discussion by sxvwuwqp** [phantom/docs#168] [Help & Support] [answered]
### Summary

'This domain is new or has not beenreviewed yet. Proceed with caution.'   Where is the audit conducted?

### Steps to Reproduce

'This domain is new or has not beenreviewed yet. Proceed with caution.'   Where is the audit conducted?

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @sxvwuwqp,

Please let us know the domain this is concerning so we may review.
